Server/routes/upload.js

```diff
@@ -11,6 +11,19 @@ import fs from 'fs'
 
 const router = express.Router();
 
+function setDate(nametype){
```

Comment:
Please change the name of the function to what it actualy do

Reply:
OK